Added `tf-ncl` Terraform wrapper to `Deployment Tools`

nhooey opened the pull request.

cyntheticfox commented on README.md:
```diff
@@ -92,6 +92,7 @@
 * [pushnix](https://github.com/arnarg/pushnix) - Simple cli utility that pushes NixOS configuration and triggers a rebuild using ssh.
 * [terraform-nixos](https://github.com/tweag/terraform-nixos) - A set of Terraform modules designed to deploy NixOS.
 * [terranix](https://terranix.org) - Use Nix and the NixOS module system to write your Terraform code.
+* [tf-ncl](https://github.com/tweag/tf-ncl] - A Terraform wrapper that reads configuration files in [Nickel](https://github.com/tweag/nickel) instead of HCL, which enables composable code+data and contracts, that is more robust and concise.
```
```suggestion
* [tf-ncl](https://github.com/tweag/tf-ncl) - A Terraform wrapper that reads configuration files in [Nickel](https://github.com/tweag/nickel) instead of HCL, which enables composable code+data and contracts, that is more robust and concise.
```
This bracket -> parenthesis change should fix the lint.

cyntheticfox commented:
Not sure this counts as a "Nix project". It's awesome, and can use Nix to build, but it isn't for writing Nix, managing NixOS, or developing Nixpkgs

cyntheticfox commented:
> Not sure this counts as a "Nix project". It's awesome, and can use Nix to build, but it isn't for writing Nix, managing NixOS, or developing Nixpkgs

Maybe it'd be fine to expand the scope to include Nickel? If nothing else, we could break it into its own category

nhooey commented:
Haha oops! I must have been in space when I added that. I was thinking about a lot of awesome Nix tools and then maybe forgot about the “Nix” part and just added something awesome. 😅
